doc/source/collections/contacts.rst

```diff
@@ -1,18 +1,24 @@
 Contacts
 ========
-a lighter version of people.  Fewer required fieldsfor capturing people who are less tightly coupled
+a lighter version of people.  Fewer required fields for capturing people who are less tightly coupled
```

Comment:
I see a few typo's being fixed but presumably these typos were being made on the auto-build and will be overwritten next time the api workflow is run.   Did you also make edits in the docstrings of the relevant module to also correct the mistake there?

Reply:
I have verified that the original typo comes from schemas.json, and the original typo has been fixed in previous lint check PR. I'm sure that in current version the api workflow follow the edited version and corrected the typo automatically here.